### DISCUSSION TITLE
Table 3, practical 3

### DISCUSSION THREAD

**Discussion by NinaGangi** [IBAHCM/RPiR#433] [Old]
Practical 3

**Comment by lynseythomson** [comment]
Hey I'm in this group :)

**Comment by IsobelAJones** [comment]
Me too

**Reply by EleonoraMilea** [reply]
Me too :)

**Comment by NinaGangi** [comment]
NinaGangi: https://github.com/IBAHCM/NinaGangiSeries03.git

**Comment by IsobelAJones** [comment]
https://github.com/IBAHCM/IsobelAJonesSeries03.git

**Comment by lynseythomson** [comment]
my repo :)
https://github.com/IBAHCM/lynseythomsonseries03.git

**Comment by EleonoraMilea** [comment]
Hi :) can you please check if code runs okay? Thank you :) 

EleonoraMilea: https://github.com/IBAHCM/EleonoraMileaSeries03.git

**Reply by EleonoraMilea** [reply]
oh, also if you can raise an issue in my repository, thank you :)

**Reply by EleonoraMilea** [reply]
Hey I kind of finished 3.2 (I can't work out how to increase variability), so could you please check if my code works well and if you can compile the HTML report? thank you :) @NinaGangi @IsobelAJones @lynseythomson

**Comment by EleonoraMilea** [comment]
Hello @NinaGangi @lynseythomson @IsobelAJones  can you please check if my 3.3 and 3.4 work fine and open an issue on my repository? Thank you 

EleonoraMilea: https://github.com/IBAHCM/EleonoraMileaSeries03.git

**Reply by EleonoraMilea** [reply]
and 3.5 too please :) @NinaGangi @lynseythomson  @IsobelAJones
